packages/gravity-ui-web/src/sass/05-components/03-organisms/_list-cards-basic.scss

```diff
@@ -0,0 +1,46 @@
+.grav-c-list-cards-basic {
+  $breakpoint-1: gravy-breakpoint(medium);
+  $breakpoint-2: gravy-breakpoint(large);
+  padding: 0;
+  list-style-type: none;
+
+  @media all and (min-width: $breakpoint-1) {
```

Comment:
Perhaps merge this `@media` block with the one below. That way we have all the non-MQ stuff first, followed by all the inside-MQ stuff. Makes it easier to read IMHO.

In this case you might as well ditch the `$breakpoint-2` var, since you'll only be used `gravy-breakpoint(medium)` once.